contract/src/main/kotlin/io/provenance/scope/loan/utility/LoanContractValidations.kt

```diff
@@ -0,0 +1,64 @@
+package io.provenance.scope.loan.utility
```

Comment:
I really like the centralization into this class.  If we ever need to locate or update these rules, they are here.
👍

Reply:
I would just note that this file doesn't contain _every_ validation rule for our contracts, mainly just ones that are going to be repeated. I would have to think about if it'd be worth taking that further step of fully extracting all validation logic to this file, but I would instinctively lean towards not doing so.